dredd_hooks.gemspec

```diff
@@ -17,6 +17,6 @@ Gem::Specification.new do |spec|
   spec.files         = Dir["{bin,doc,lib}/**/*", "CHANGELOG.md", "Gemfile", "LICENSE.txt", "Rakefile", "README.md" ]
   spec.test_files    = Dir["{features,spec}/**/*"]
 
-  spec.add_development_dependency "rake", "~> 12.3"
+  spec.add_development_dependency "rake", "~> 13.0"
```

Comment:
The change that is non-backwards-compatible is that support for Ruby 2.2 was dropped. Considering that [Ruby 2.2 is no longer supported](https://www.ruby-lang.org/en/news/2018/06/20/support-of-ruby-2-2-has-ended/), I think that this is perfectly okay. :shipit:

Reply:
The range could also be written with `>= 12.3, < 14.0` or similiar too to avoid unnessecerily dropping support. Although, I have no real opinion on which to take, just sharing the options.

Reply:
That is a good point @kylef.

Because rake is a _development_ dependency, I don't think we have much to "support" in this case, but it certainly wouldn't hurt to keep a broader range of versions if it feels safer.

In that case, as you said,  `>= 12.3, < 14` is correct :+1: (`14` and `14.0` are equivalent).